### DISCUSSION TITLE
[Bug] Fail to start a standalone Pulsar(v3.0.0) with local memory as metadata store

### DISCUSSION THREAD

**Discussion by Richelieu42** [apache/pulsar#20380] [Q&A]
![1](https://github.com/apache/pulsar/assets/115224697/952ebbd8-dea1-4bb1-80e6-e7eedf62dc5e)
<img width="860" alt="2" src="https://github.com/apache/pulsar/assets/115224697/aab48faf-c90a-4f0f-bedb-6521dfd991ac">
<img width="881" alt="3" src="https://github.com/apache/pulsar/assets/115224697/4d1c62d8-71c3-481c-83cc-8e044e9a329d">
<img width="1082" alt="4" src="https://github.com/apache/pulsar/assets/115224697/f91471c0-736a-4607-985a-dbb7522fdaa7">

**Comment by Richelieu42** [comment]
Version 2.11.0 has the same problem.

**Comment by hangc0276** [comment]
It looks like a documentation bug. If you are interested in the part, welcome to fix it.
